content/SCALE/SCALETutorials/Credentials/ManageLocalUsersSCALE.md

```diff
@@ -46,90 +46,91 @@ Enter the name you want to use for the administrator account.
+When the path does not end with a sub-directory matching the username, TrueNAS creates a new sub-directory if the **Create Home Directory** checkbox is enabled.
 TrueNAS shows the path to the user home directory when editing a user.
 
-You can set the home directory permissions directly under the file browser. 
-You cannot change TrueNAS default user account permissions.
+Select the **Home Directory Permissions** (**Read**, **Write**, **Execute**) for each (**User**, **Group**, **Other**) to set Unix permissions for the user home directory. Built-in users are read-only and can not modify these permissions settings.
 
 ### Configuring Authentication Settings
 
-{{< trueimage src="/images/SCALE/22.12/AddUserHomeDirAuthSCALE.png" alt="Add User Home Directory and Authentication Settings" id="6: Add User Home Directory and Authentication Settings" >}}
+{{< trueimage src="/images/SCALE/23.10/AddUserHomeDirAuthSCALE.png" alt="Add User Home Directory and Authentication Settings" id="6: Add User Home Directory and Authentication Settings" >}}
 
 You can assign a public SSH key to a user for key-based authentication by entering or pasting the *public* key into the **Authorized Keys** field.
 
 {{< hint type=important >}}
 Do *not* paste the private key.
 {{< /hint >}}
 
+You can also click **Choose File** under **Upload SSH Key** and browse to the location of an SSH key file.
+
 If you are using an SSH public key, always keep a backup of the key.
 
-Select the [shell]({{< relref "LocalUsersScreensSCALE.md" >}}) option for the user from the **Shell** dropdown options. 
-Set the local admin user shell to **TrueNAS CLI** to open shell in the TrueNAS CLI. Set to **TrueNAS Console** to open in the Console Setup Menu for SCALE. 
+Select the [shell]({{< relref "LocalUsersScreensSCALE.md" >}}) option for the user from the **Shell** dropdown options.
+Options are **nologin**, **bash**, **rbash**, **dash**, **sh**, **tmux**, and **zsh**.
+Admin users can also use **TrueNAS CLI** to open shell in the TrueNAS CLI and **TrueNAS Console** to open in the Console Setup Menu for SCALE.
 
 Selecting **Lock User** disables all password-based functionality for the account until you clear the checkbox.
 
-**Allowed sudo commands**, **Allow all sudo commands**, **Allowed sudo commands with no password** and **Allow all sudo commands with no password** allows the account to act as the system administrator using the [sudo](https://www.sudo.ws/) command. Leave it disabled for better security.
+**Allowed sudo commands**, **Allow all sudo commands**, **Allowed sudo commands with no password** and **Allow all sudo commands with no password** allow the account to act as root using the [sudo](https://www.sudo.ws/) command.
+To allow sudo commands, use **Allowed sudo commands** or **Allowed sudo commands with no password** to list the executable path(s) to specific sudo commands allowed for this user.
```

Comment:
Okay, this one bothers me a bit, are you saying the command lists the executables or are you saying they can enter the executable paths to command?  This will be very hard for a non-native speaker to understand. Use simple English here. If what they gain is the ability to enter commands or paths say that in as simple a way as possible.